src/app/loans/loans-routing.module.ts

```diff
@@ -205,7 +207,10 @@ const routes: Routes = [
           data: { title: extract('Edit Loans Account'), breadcrumb: 'Edit Loans Account', routeParamBreadcrumb: 'Edit' },
           component: EditLoansAccountComponent,
           resolve: {
-            loansAccountAndTemplate: LoansAccountAndTemplateResolver
+            loansAccountAndTemplate: LoansAccountAndTemplateResolver,
+            actionButtonData: LoanActionButtonResolver,
+            loanTemplate: LoanApprovalTemplateResolver,
+            loanData: LoanDetailsGeneralResolver
```

Comment:
I think you have mistakenly made the changes in the `edit-loans` routing instead of `actions/:actions`. Please fix this.

Reply:
Also, I don't find the required property 'approve' in the `loan-account-actions.component.ts` file, please add it.

Reply:
I'll fix this

Reply:
> I think you have mistakenly made the changes in the `edit-loans` routing instead of `actions/:actions`. Please fix this.

I think edit loans has not been fixed by me. Anyway shall I remove the routing edit-loans-account ?

Reply:
Only remove the lines added by you.

Reply:
Is this /loan/{loanId}/template API needed in loan approval component ?

Reply:
And can you suggest me a loan to test this PR ?

Reply:
https://staging.mifos.io/#/viewloanaccount/2

Reply:
> Is this /loan/{loanId}/template API needed in loan approval component ?

Yes its required.